index.bs

```diff
@@ -229,6 +254,9 @@ spec: fetch; urlPrefix: https://fetch.spec.whatwg.org/
 
         If no such context exists, throw an "{{InvalidStateError}}" {{DOMException}}.
 
+    1. If the [=portal state=] of |predecessorBrowsingContext| is not *none*,
+        throw an "{{InvalidStateError}}" {{DOMException}}.
```

Comment:
Let's add some non-normative text that this means that nested portals can't be activated.